### DISCUSSION TITLE
Keep time range when traveling to home dashboard by clicking Grafana logo

### DISCUSSION THREAD

**Discussion by vvandel-nfir** [grafana/grafana#41677] [Dashboards, Panels & Visualizations]
Similar to how dashboard links have the option `"keepTime": true`, I would like to have this feature when traveling to the home dashboard by clicking on the Grafana logo.

A decent workaround is to add the home dashboard as a dashboard link, which _is_ able to keep the time range selection.

However, I think it would be a quality of life improvement to also have this feature for the innate link to the home dashboard in the Grafana logo. This option could be configurable in the grafana.ini.

**Comment by anavss** [comment]
Hi @vvandel-nfir 👋🏼 our product designers are investigating usability improvements to Grafana's main navigation. I spotted your request and wondered if you would be interested in talking to them? You can [apply to participate here](https://ethn.io/32993).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
